src/components/following/FollowingFollowerList.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div
+    id="scroll-target"
+    class="post-container"
+    v-if="actors && actors.length > 0"
+  >
+    <v-list>
+      <v-list-item-group color="primary">
+        <v-list-item
+          class="actor-short-list"
+          v-for="(actor, index) in actors"
+          v-intersect="onIntersect"
+          :key="index"
+          :data-index="index"
+          @click="detail(actor)"
+        >
+          <v-list-item-content>
+            <SummarizedActor :actor="actor" :noSummary="true" />
+          </v-list-item-content>
+        </v-list-item>
+      </v-list-item-group>
+    </v-list>
+    <v-progress-linear
+      indeterminate
+      height="15"
+      absolute
+      bottom
+      :light="$vuetify.theme.dark"
+      :dark="!$vuetify.theme.dark"
+      :active="isLoading"
+    >
+      <template v-slot>
+        <strong class="caption">loading more...</strong>
+      </template></v-progress-linear
+    >
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Emit } from "vue-property-decorator";
+import { ActivityObject, Link } from "activitypub-objects";
+
+import FollowingActor from "@/components/following/FollowingActor.vue";
+import SummarizedActor from "@/components/actor/ActorSummarized.vue";
+import { User } from "@/model/user";
+
+@Component({
+  components: {
+    FollowingActor,
+    SummarizedActor,
+  },
+})
+export default class FollowingFollowerList extends Vue {
```

Comment:
I guess the naming was initially from me? But now I'm really confused about the "FollowingFollowerList"... isn't it just the `FollowingList`?

Reply:
Ah, more like ActorList. Because it can show Followers, Following or any list of Actors. And then I put it in the actor folder. What do you think about?

Reply:
Sounds good!